internals/templates/setup/ensureIntlSupport.js

```diff
@@ -0,0 +1,10 @@
+export default function ensureIntlSupport() {
+  if (window.Intl) return Promise.resolve();
+  return new Promise((resolve) => {
+    resolve(System.import('intl'));
```

Comment:
`System.import` is deprecated now, isn't it?
Should we use `import` instead?

https://github.com/webpack/webpack/issues/2163